### PR TITLE
[client] Fixing docker build errors from renaming to cli

### PR DESCRIPTION
**Pull request by sunmilee**
## Motivation

Fixing the naming in docker build files after this change https://github.com/libra/libra/pull/2319

## Test Plan
docker/validator/build.sh

## Related PRs
https://github.com/libra/libra/pull/2319

### CODE REVIEW

File: docker/client/client.Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ FROM toolchain AS builder
 
 COPY . /libra
 
-RUN cargo build --release -p libra-node -p client -p config-builder && cd target/release && rm -r build deps incremental
+RUN cargo build --release -p libra-node -p cli -p config-builder && cd target/release && rm -r build deps incremental
 RUN strip target/release/client
```
Comment (sausagee) [inline docker/client/client.Dockerfile]: I think should be ` RUN strip target/release/cli`.

**Comment by thefallentree** [comment]
@sunmilee  I fixed it

**Pull request closed without merging**